src/seedu/addressbook/logic/Logic.java

```diff
@@ -96,4 +96,9 @@ private void recordResult(CommandResult result) {
             lastShownList = personList.get();
         }
     }
+    private void saveCommand(Command command) throws Exception{
```

Comment:
Does not seem like a good idea to name it `saveCommand`, since `____Command` has been established as commands the user of the application uses.